README.md

```diff
@@ -20,105 +21,96 @@ https://puppet.com/docs/pe/latest/getting_support_for_pe.html#pe-support-script
-mod 'tkishel-pe_tune',
-  :git => 'https://github.com/tkishel/pe_tune.git',
-  :branch => 'master'
-EOF
-
-bolt puppetfile install
-```
+The preferred method of installation of `pe_tech_check` is via Code manager however should this not be possible other options are available
 
 #### Manually
 
-To manually install the `pe_tech_check` module, execute the following command on the Primary Master.
+To manually install the `pe_tech_check` module, execute the following command on the Primary Server.
 
 ```bash
 puppet module install puppetlabs-pe_tech_check --modulepath=/opt/puppetlabs/puppet/modules
 ```
 
 Doing so will install this module into the base module path, making its tasks available without interfering with other modules.
 
-If your Primary Master has environment caching enabled (which is true by default if Code Manager is being used), flush the environment cache to enable the tasks in this module by running the following command on the Primary Master:
+If your Primary Server has environment caching enabled (which is true by default if Code Manager is being used), flush the environment cache to enable the tasks in this module by running the following command on the Primary Server:
 
 ```bash
 /opt/puppetlabs/puppet/modules/pe_tech_check/scripts/flush_environment_cache.sh
 ```
 
-## Usage
 
-### Run the `pe_tech_check::configure` task
 
-#### Via Bolt
 
-```bash
-bolt task run pe_tech_check::configure --nodes <master_fqdn>
-```
+#### Using Bolt
 
-#### Via the Console
+Bolt can be used either on the Primary Server  or another machine with connectivity to the Primary Server. Firstly, follow the instruction for [installing Bolt](https://puppet.com/docs/bolt/latest/bolt_installing.html).
 
-In the Console, run the `pe_tech_check::configure` task, targeting the Primary Master.
+The following block of code will create a Bolt Project under the logged in user's home directory and set up the module:
 
-#### Via Puppet Task
+```bash
+mkdir -p ~/tech_check
+cd ~/tech_check
 
-From the command line of the Primary Master, run:
+cat >>Puppetfile <<EOF
+mod 'puppetlabs-stdlib'
+mod 'puppetlabs-pe_tech_check'
+EOF
 
-```bash
-puppet task run pe_tech_check::configure --nodes $(puppet config print certname)
+bolt project init
+bolt module install --no-resolve
 ```
 
-#### Task parameters
 
-##### `install_pe_metrics` (Boolean, default: true)
+## Usage
+
+Pe_tech_check consumes Metric Provided by Puppet_Metrics_Collector Module, Puppet Enterprise  >= 2019.8.8 and >= 2021.3 Come with Puppet Metrics Collector built in however for PE 2019.8.8 this feature must be enabled before proceeding.
 
-Temporarily install and configure the `puppet_metrics_collector` module, if it is not already installed.
+To enable metrics collector in 2019.8.8 set the following parameters to true:
 
-https://forge.puppet.com/puppetlabs/puppet_metrics_collector
+```
+puppet_enterprise::enable_metrics_collection
+puppet_enterprise::enable_system_metrics_collection
```

Comment:
Could we have a bit more information here for a reader who may or may-not know how or where to set these parameters?  For example, is there a link that describes where in the Console or in hiera to set these? 

For example, one way to do this would be to update your ``/etc/puppetlabs/code/environments/production/data/common.yaml`` something like:

```diff
45a46,50
> 
> # enable metrics collection
> puppet_enterprise::enable_metrics_collection: true
> puppet_enterprise::enable_system_metrics_collection: true
> 
[root@pe-server-60d9bb-0]~/tech_check# 
```

Or update on the console and commit the change:

![image](https://user-images.githubusercontent.com/16652757/140973528-06bd8426-bd79-441b-8ce3-697713fbabc6.jpeg)

Also, might be good to remind the user to run ``puppet agent -t`` on the primary to update the server